bakerydemo/base/management/commands/reset_demo.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+from django.core.files.storage import default_storage
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from wagtail.documents import get_document_model
+from wagtail.images import get_image_model
+
+
+class Command(BaseCommand):
+    def handle(self, **options):
+        # 1. (optional) Remove all objects from S3
+        if "s3" in settings.DEFAULT_FILE_STORAGE:
+            self.stdout.write("Removing files from S3")
+            default_storage.bucket.objects.all().delete()
+        else:
+            self.stdout.write("Removing images")
+            get_image_model().objects.all().delete()
+
+            self.stdout.write("Removing documents")
+            get_document_model().objects.all().delete()
+
+        # 2. Reset database to nothing
+        self.stdout.write("Reset schema")
+        call_command("reset_schema", interactive=False)
+
+        # 3. Rebuild database
+        call_command("migrate", interactive=False)
+
+        # 4. Re-import data
+        call_command("load_initial_data")
```

Comment:
Should we also call `update_index`? For the search index?

Reply:
@allcaps This happens as part of `load_initial_data`, so isn't needed as a separate step here.

I'd suggest that this might also be a good place to change the password of the admin user to something other than `changeme` - or set up a new superuser account - so that people trying the demo don't take this as an invitation to change the password and lock out everyone else.

Reply:
My intention is to attempt to either prevent the password from being changed, or have a recurring task which forces the password back. I suspect as part of this we'll have a few different users with different permission levels created by default to allow better testing.

Reply:
There's now a `reset_admin_password` command which achieves this.